README.md

```diff
@@ -20,3 +20,7 @@ A collaborative list of AITU student's projects. To add your own project below,
 - [Tamerlan Satualdypov/CovidKazakhstan](https://github.com/onl1ner/CovidKazakhstan) – iOS application to track number of coronavirus COVID-19 cases in Kazakhstan.
 - [Tamerlan Satualdypov/Hands](https://github.com/onl1ner/Hands) – memory game with hand gesture recognition that will keep you brain in a good shape!
 - [Tamerlan Satualdypov/STTextView](https://github.com/onl1ner/STTextView) - a light-weight Swift library that adds a placeholder to the UITextView.
+
+
+## Telecommunication Systems
+- [Alibek Mombekov/SpeechRecognition](https://github.com/olimeme/speechrecognition) – speech recognition system written on python.
```

Comment:
The word "python" at the end of your sentence should be capitalized, as this is the name of the programming language you used